plugins/catalog-react/src/components/EntityRefLink/humanize.test.ts

```diff
@@ -34,6 +34,23 @@ describe('humanizeEntityRef', () => {
     expect(title).toEqual('component:software');
   });
 
+  it('formats entity in default namespace without skipping default namespace', () => {
+    const entity = {
+      apiVersion: 'v1',
+      kind: 'Component',
+      metadata: {
+        name: 'software',
+      },
+      spec: {
+        owner: 'guest',
+        type: 'service',
+        lifecycle: 'production',
+      },
+    };
+    const title = humanizeEntityRef(entity, { defaultNamespace: false });
```

Comment:
What happens if you pass `true`? 😁 I think we need to handle that somehow, perhaps even disallow it by making the type `string | false`

Reply:
Ou, you have actually told me to have it like that. Sorry. New commit done.